### ISSUE TITLE
client.on('friendRelationShip') does not detect friend requests

### ISSUE THREAD

**Issue by Fall1nglight** [DoctorMcKay/node-steam-user#374]
## Describe the bug
Hi everyone,
I have been experiencing that the script does not detect friend requests. Maybe I missed something. It would be grateful if you could help me out 👍 

## My code
```
client.on('friendRelationShip', (steamId, relationship) => {
  if (
    relationship === SteamUser.EFriendRelationship.RequestRecipient ||
    relationship === SteamUser.EFriendRelationship.RequestInitiator
  ) {
    client.addFriend(steamId, (err, personaName) => {
      if (err) console.error(err);
      console.log(personaName);
    });

    client.chat.sendFriendMessage(steamId, 'Hi!', (err) => {
      if (err) console.error(err);
    });
  }
`});`
```

## Versions
Node: v16.2.0
steam-totp: 2.1.1
steam-tradeoffer-manager: 2.10.3
steam-user: 4.19.12
steamcommunity: 3.43.1

**Comment by DoctorMcKay** [comment]
The event name is `friendRelationship`, not `friendRelationShip`. Event names are case sensitive.

**Comment by Fall1nglight** [comment]
Thank you. I don't know if it's a bug, but vscode suggested me the event name.
![bug](https://user-images.githubusercontent.com/80258380/134480658-8a753b1e-8647-423e-a2be-75f4ab93c9d5.png)


> The event name is `friendRelationship`, not `friendRelationShip`. Event names are case sensitive.

**Comment by Jmopel** [comment]
It will be fixed soon: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55948

**Comment by Jmopel** [comment]
@Fall1nglight, Done. It's fixed in `@types/steam-user@4.19.3`
### For `TypeScript`
Install it in the local `devDependencies` using your favorite package manager:
```ps1
pnpm i -D @types/steam-user@^4.19.3
npm i -D @types/steam-user@^4.19.3
yarn add -D @types/steam-user@^4.19.3
```
### For `JavaScript` in VSCode with enabled `Automatic Type Acquisition`
The `package.json` and `node_modules` where you need to update it is located here:
```ps1
# On Windows
C:\Users\<user>\AppData\Local\Microsoft\TypeScript\4.4
```
Just use `update` there.
